cmd_cmd.go

```diff
@@ -91,10 +103,14 @@ will be launched via "bash -c" using "exec".`,
 	return execCmd
 }
 
-func runCommand(subcmd string) error {
+func runCommand(subcmd string, prop *propagation.PropagationContext) error {
 	fmt.Println("running /bin/bash -c", subcmd)
 	cmd := exec.Command("/bin/bash", "-c", subcmd)
 
+	cmd.Env = append(os.Environ(),
+		"HTTP_X_HONEYCOMB_TRACE=" + propagation.MarshalHoneycombTraceContext(prop),
```

Comment:
I'm not sure this is right thing to use either, but then also I don't have an alternative suggestion.

This env var is only used in beeline-ruby: https://github.com/search?q=org%3Ahoneycombio+HTTP_X_HONEYCOMB_TRACE&type=code.

Reply:
I've changed this to only `HONEYCOMB_TRACE`. This matches the other `HONEYCOMB_*` environment variables as well as the `X-Honeycomb-Trace` HTTP header.

I should also add a bit of docs around that, I guess.

Reply:
It might be worth noting that the `HTTP_X_HONEYCOMB_TRACE` is not an environment variable in the ruby beeline, it's only pulled from the rack environment